client/lib/posts/actions.js

```diff
@@ -205,13 +202,14 @@ PostActions = {
 						type: 'RECEIVE_POST_AUTOSAVE',
 						error: error,
 						autosave: data,
+						site,
 					} );
 
 					callback( error, data );
 				}
 			);
 		} else {
-			PostActions.saveEdited( null, null, callback, { recordSaveEvent: false, autosave: true } );
```

Comment:
We're losing `autosave: true` here. Is this intended?

Reply:
Nice catch 👍 It was not intended and happened during the rebase as the version I had before did not contained the flag. It was corrected.